### DISCUSSION TITLE
Upgrade to Svelte 4

### DISCUSSION THREAD

**Discussion by oktapascal** [inertiajs/inertia#1600] [Ideas]
Svelte 4 has been released, does inertia/svelte support upgrading from Svelte 3 to Svelte 4?

**Comment by reinink** [comment]
We just added support for Svelte 4 tonight 👍

https://github.com/inertiajs/inertia/releases/tag/v1.0.9

**Comment by oktapascal** [comment]
Cool, Thank you for all your work
